site/content/blog/2021-06-01-whats-new-in-svelte-june-2021.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Svelte: June 2021
+description: Progress on SvelteKit V1 and tighter TypeScript/Svelte integrations in language tools
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, we saw lots of contributions to SvelteKit and its docs. The language tools also got some new features alongside some hydration bug fixes in Svelte core. Let's jump into the updates...
+
+## New in SvelteKit
+- `svelte.config.js` config files can now be loaded in ESM format (`.js` instead of `.cjs` - either can be used). It is recommended to use the `.js` format going forward.
+- AMP pages will now use the rendered CSS, rather than emitted CSS
+- `trailingSlash: 'never' | 'always' | 'ignore'` is now available in the config. This should make it easier to build sites that work with static hosting providers that expect a trailing slash for `index.html` pages, and provides an escape hatch for anyone that needs more complex behaviour.
+- The router will no longer intercept navigation for URLs that the app does not own. This fixes a crash for apps that have `<a>` elements on the page with the same origin but don't share a base path with the app.
+- Hash only changes are now handled by the router - fixing the browser's "back" navigation between hash changes.
+- Svelte Kit builds and dev versions should now work on Windows (`pnpm build` and `dev` commands were failing due to lack of an `rm` command)
```

Comment:
minor wording suggestion
```suggestion
- Svelte Kit builds and dev versions should now work on Windows (`pnpm build` and `dev` commands were failing due to usage of the `rm` command)
```

Reply:
Is this even worthy to highlight here? AFAIK this is only a problem when working _on_ Kit, not _with_ Kit.

Reply:
Yeah probably not. I'll remove it :)